### PR TITLE
Add universions recipe

### PR DESCRIPTION
**Pull request by fabiencelier**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/universions/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set name = "universions" %}
+{% set version = "0.1.0.dev201911021553" %}
```
Comment (xhochy) [inline recipes/universions/meta.yaml]: Is there a release version available?

Comment (fabiencelier) [reply recipes/universions/meta.yaml]: Not yet, is this mandatory ?
I was trying to figure out how to publish to conda-forge before releasing to check that I would be able to publish but I can do a release if that's better

Comment (xhochy) [reply recipes/universions/meta.yaml]: Everything on conda-forge is considered a release, there is work on enabling pre-release builds but that currently isn't here. Tests are green and recipe looks good, so make a release once ready and we will immediately merge the PR (feel free to ping `@conda-forge/staged-recipes` then) with the released version.

Comment (fabiencelier) [reply recipes/universions/meta.yaml]: ok I'll do that !
Thank you for the answer

**Comment by fabiencelier** [comment]
@conda-forge/staged-recipes 
I have updated with a release version and added an entry_point for a command line

**Comment by fabiencelier** [comment]
:tada: 

Thanks guys !

**Pull request merged**